python/GangaLHCb/test/GPI/TestDataFiles.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import
+
+from Ganga.testlib import external
+from Ganga.testlib.GangaUnitTest import GangaUnitTest
+
+
+class TestDataFiles(GangaUnitTest):
+
+    @external
+    def testDataFiles(self):
+
+        from Ganga.GPI import DiracFile, config
+
+        # LFNs
+        name = 'test.txt'
+        lfn = DiracFile(lfn=name)
+        assert lfn.lfn == name
+        lfn = DiracFile('lfn:'+name)
+        assert lfn.lfn == name
+        lfn = DiracFile('LFN:'+name)
+        assert lfn.lfn == name
+        try:
+            lfn = DiracFile('pfn:'+name)
+            raise RuntimeError('Should have got exception trying to create DiracFile from pfn.')
+        except:
+            pass
```

Comment:
This could be changed to use [`pytest.raises`](https://pytest.org/latest/assert.html#assertions-about-expected-exceptions) to be slightly neater. Not a big deal, more informing for the future.